5617-LazyListMorph-missing-theme-hook

Ducasse opened the pull request:
Fixes: #5617 make sure that we can control the color of non/selected list item.

Ducasse committed:
Fixes: #5617 make sure that we can control the color of non/selected list item.

Ducasse commented:
Si vous pouvez jeter un oeil pour regarder si cela ne casse pas vos themes
Moi j'en ai besoin pour les miens.

estebanlm commented on src/Polymorph-Widgets/UITheme.class.st:
```diff
@@ -5321,6 +5321,13 @@ UITheme >> selectedFillStyleFor: aMorph height: anInteger [
+UITheme >> selectedItemListTextColor [
+	"Answer the color for the selected item in a list text."
+
+	^ Color black
```
default selected color needs to be "textColor", not black.

Ducasse replied:
Ok I will check if it is working with my style. My problem is that I could not see the text of certain items. 
Thanks for the feedback

Ducasse replied:
This does not work for me. 


![Screenshot 2020-02-01 at 11 33 58](https://user-images.githubusercontent.com/432549/73590786-c2d30e80-44e6-11ea-8a87-b95a3819b324.png)

Ducasse replied:
This is the only place in the system it does not work. 
I do not understand why this is happening. In particular since

In my theme I redefined textColor this way. 

	"This is only for all the list and the text except that the selection color can hide it on certain list."
	^ Color black

Ducasse replied:
I'm investigating (but my brain is fucked by the flu) still I spent a lot of time on this issue in the past. There is something strange.

Ducasse replied:
Now I'm totally confused :( So probably should go to bed. ;(

Ducasse replied:
Ok apparently lazyListMorph is used in dropBox. Not the finder tree.

Ducasse replied:
So for the dropbox what you propose is working.

Ducasse commented:
Now I have a problem with the tree color or something like that.

Ducasse commented:
I do not get why the second change I did does not show up. 
So boring.

Ducasse committed:
Update UITheme.class.st

did it manually. No idea why my changes did not arrive. May be I did not push it.

Ducasse commented:
The change was done!
